docs/cookbook/syncing/kyoto.md

```diff
@@ -0,0 +1,88 @@
+# Sync a wallet with Kyoto
+
+!!! tip 
+    This page is up-to-date with version `1.0.0-beta.5` of bdk.
+
+[BIP157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki) and [BIP158](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki) define a protocol for light clients to sync with the Bitcoin network without downloading the entire set of blocks in the chain of most work. These proposals define _compact block filters_, which allow a client to download a small commitment
+for the scripts contained in each block. These commitments, or filters, may be checked for inclusion of scripts owned by a user. In the event of a match, the light client may download and verify a block indeed contains a relevant transaction. Syncing via compact block filters offers privacy advantages over other chain sources, as the nodes serving the blocks to the client are only aware that the client is interested in an entire block, which may contain thousands of transactions.
+
+One such implementation of this protocol is [Kyoto](https://github.com/rustaceanrob/kyoto), which is a node and client for compact block filter based syncing. The [`bdk_kyoto`](https://github.com/bitcoindevkit/bdk-kyoto) crate supports an integration between Kyoto and `bdk_wallet`, so developers using `bdk_wallet` have a simple option to provide privacy-preserving and memory-conservative wallet syncing for their users.
+
+The following example uses the `bdk_kyoto` crate to recover and update a `bdk_wallet` using compact block filters.
+
+### Add required bdk dependencies to your `Cargo.toml` file
+
+```toml
+bdk_kyoto = { version = "0.4.0", default-features = false, features = ["rusqlite", "wallet", "callbacks"] }
+bdk_wallet = { version = "1.0.0-beta.5" }
+tokio = { version = "1.37", features = ["full"], default-features = false }
+```
+
+### Create and sync the wallet
+
+```rust
```

Comment:
looks like this is the same code as in the actual code file correct? if so then you can pull the code using a code snippet (see other pages in book like quickstart) @rustaceanrob